docs/cudf/source/_ext/PandasCompat.py

```diff
@@ -3,10 +3,18 @@
 # This file is adapted from official sphinx tutorial for `todo` extension:
 # https://www.sphinx-doc.org/en/master/development/tutorials/todo.html
 
+import functools
+import operator
+from typing import cast
+
 from docutils import nodes
+from docutils.nodes import Element
 from docutils.parsers.rst import Directive
-from sphinx.locale import get_translation
-from sphinx.util.docutils import SphinxDirective
+from sphinx import addnodes
+from sphinx.domains import Domain
+from sphinx.errors import NoUri
+from sphinx.locale import _, get_translation
```

Comment:
What does the `_` do here? Typically `_` is used as a placeholder, so I worry that this name will be overwritten or misunderstood by readers. Can we import this as some other name, like `from sphinx.locale import _ as ...` so it's possible to find this?

It looks like it's defined as `_ = get_translation('sphinx')`. Maybe we can use the name `get_translation_sphinx`.

Reply:
Sounds good. `_` is a horrible name for an object